cloudomate/test/test_captchasolver.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import requests
+import json
+import os
+import base64
+import time
+
+from cloudomate.util.captchasolver import CaptchaSolver, ReCaptchaSolver
+from mock import patch, MagicMock
+
+
+class TestCaptchaSolver(unittest.TestCase):
+
+    def setUp(self):
+        self.captcha_solver = CaptchaSolver("213389asd8u912823")
+        self.recaptcha_solver = ReCaptchaSolver("213389asd8u912824")
+
+    @patch("json.loads")
+    @patch("requests.post")
+    def test_get_balance(self, mock_requests, mock_json):
+        self.captcha_solver.get_balance()  
+
+
+          
+    def test_get_current_key(self):
+        self.assertEqual(self.captcha_solver.get_current_key(),"213389asd8u912823")
+        self.assertEqual(self.recaptcha_solver.get_current_key(),"213389asd8u912824")
+
+    @patch('time.sleep')
+    @patch('base64.b64encode')
+    @patch('builtins.open')
+    @patch('os.path.isfile')
```

Comment:
I'm not sure whether mocking of os-level methods is a good idea. Do you really need to mock these?